app/views/support_interface/provider_users/new.html.erb

```diff
@@ -23,6 +23,9 @@
 
       <%= f.govuk_text_field :email_address, label: { text: 'Email address' } %>
       <%= f.govuk_text_field :dfe_sign_in_uid, label: { text: 'DfE Sign-in UID' } %>
```

Comment:
The `hint_text:` option might be a good place for the new copy?

https://govuk-form-builder.netlify.com/form-elements/text-field/